modules/gnome-extensions/README.md

```diff
@@ -0,0 +1,30 @@
+# `gnome-extensions`
+
+The `gnome-extensions` module can be used to install Gnome extensions inside system directory.
+
+This module is universally compatible with all distributions which ship Gnome, as it's not tied to specific distribution packaging format for installing extensions.  
+
+Almost all Gnome extensions are compatible for installation.  
+Only rare intervention that might be needed is for extensions which require some additional system dependencies, like Pano.   
```

Comment:
I got rid of "Pano", in case it gets depreciates, so there is no need to edit the README for that.

I added the explanation in Usage, as I find it fits better there.